### PR TITLE
Release version 1.1.0

### LINKED ISSUE #54: Release version 1.1.0
### Describe your question

This issue is for tracking changes for the 1.1.0 release.  Target release date: **8 December 2021**.

### [Release steps](https://github.com/10up/retro-winamp-block/blob/develop/CONTRIBUTING.md#release-instructions)
- [x] Branch: Starting from `develop`, cut a release branch named `release/1.1.0` for your changes.
- [x] Version bump: Bump the version number in `package-lock.json`, `package.json`, `readme.txt`, and `retro-webamp-block.php` if it does not already reflect the version being released.
- [x] Changelog: Add/update the changelog in `CHANGELOG.md` and `readme.txt`.
- [x] Props: update `CREDITS.md` file with any new contributors, confirm maintainers are accurate.
- [x] New files: Check to be sure any new files/paths that are unnecessary in the production version are included in `.distignore`.
- [x] Readme updates: Make any other readme changes as necessary. `README.md` is geared toward GitHub and `readme.txt` contains WordPress.org-specific content.  The two are slightly different.
- [x] Merge: Make a non-fast-forward merge from your release branch to `develop` (or merge the pull request), then do the same for `develop` into `trunk` (`git checkout trunk && git merge --no-ff develop`).  `trunk` contains the stable development version.
- [x] Push: Push your `trunk` branch to GitHub (e.g. `git push origin trunk`).
- [x] Release: Create a [new release](https://github.com/10up/retro-winamp-block/releases/new), naming the tag and the release with the new version number, and targeting the `trunk` branch.  Paste the changelog from `CHANGELOG.md` into the body of the release and include a link to the closed issues on the [milestone](https://github.com/10up/retro-winamp-block/milestone/4?closed=1).
- [x] SVN: Wait for the [GitHub Action](https://github.com/10up/retro-winamp-block/actions/workflows/wordpress-plugin-deploy.yml) to finish deploying to the WordPress.org repository.  If all goes well, users with SVN commit access for that plugin will receive an emailed diff of changes.
- [x] Check WordPress.org: Ensure that the changes are live on https://wordpress.org/plugins/retro-winamp-block/.  This may take a few minutes.
- [x] Close milestone: Edit the [milestone](https://github.com/10up/retro-winamp-block/milestone/4) with release date (in the `Due date (optional)` field) and link to GitHub release (in the `Description` field), then close the milestone.
- [x] Punt incomplete items: If any open issues or PRs which were milestoned for `1.1.0` do not make it into the release, update their milestone to`1.2.0` or `Future Release`.

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

### PR DESCRIPTION
**Pull request by Sidsector9**
Closes #54

### CODE REVIEW

File: retro-webamp-block.php [hunk 1]
```diff
@@ -3,7 +3,7 @@
  * Plugin Name:       Retro Winamp Block
  * Plugin URI:        https://wordpress.org/plugins/retro-winamp-block/
  * Description:       A Winamp-styled audio block for all your retro music player needs.
- * Version:           1.0.1
+ * Version:           1.1.0
  * Requires at least: 5.8
  * Requires PHP:      7.2
```
Comment (jeffpaul) [inline retro-webamp-block.php]: Should this get changed to 5.6?

**Comment by jeffpaul** [comment]
@Sidsector9 curious if we should be committing composer-lock.json to the repo like in other projects?  Also whether this file structure looks correct to you?
Screen Shot 2022-06-14 at 1 09 14 PM

**Comment by Sidsector9** [comment]
@jeffpaul answering some of the questions:

> curious if we should be committing composer-lock.json

Yes. It'll just ensure everyone developing on the project are on the same version.

> Also whether this file structure looks correct to you?

If you're referring to the `/build` directory, that structure is something generated by `wp-scripts`. Not 100% sure how, but `wp-scripts` does offer an option to override config through `webpack.config.js`. If so, we can standardize the output across all projects. I'll look into this next week.

File: retro-webamp-block.php (hunk 1 above)
Comment (Sidsector9) [reply retro-webamp-block.php]: @jeffpaul the plugin breaks on 5.6 & 5.7. It only worked when WP was at minimum 5.8

**Review by jeffpaul — DISMISSED** [review]

File: retro-webamp-block.php (hunk 1 above)
Comment (jeffpaul) [reply retro-webamp-block.php]: Sorry, was referencing the PHP minimum and not WP minimum.  The PHP min in readme.txt and README.md both show PHP>=5.6 whereas here we show 7.2.

**Review by jeffpaul — APPROVED** [review]

**Pull request merged**